ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -360,6 +389,16 @@ export default class AlertTableRow extends React.Component {
     const noiseProfileTooltip = alert.noise_profile
       ? noiseProfiles[alert.noise_profile.replace('/', '')]
       : noiseProfiles.NA;
+    const showSideBySideLink =
+      alert.series_signature.framework_id === browsertime &&
+      !alert.series_signature.tags.includes('interactive') &&
+      !browsertimeBenchmarksTests.includes(alert.series_signature.suite) &&
+      alert.backfill_record &&
+      (alert.backfill_record.status ===
```

Comment:
You could also use includes() for this check if you create an array for the statuses your interested in.
```suggestion
      (backfillResultStatuses.includes(alert.backfill_record.status)
```
Why are you interested in only these three statuses?

Reply:
I discovered just before I updated the PR that the statuses that are past the point the `side-by-side` jobs were triggered are just `backfilled`, `successful`, and `failed`.

Reply:
@alexandru-io Could you also update the logic as mentioned above in the suggestion?